apps/api/src/app/app.module.ts

```diff
@@ -268,6 +272,7 @@ const autoSchemaFile = environment.production
       load: [
         AdrAndMachineLicenseClientConfig,
         AirDiscountSchemeClientConfig,
+        ConsultationPortalClientConfig,
```

Comment:
Need to configure CONSULTATION_PORTAL_CLIENT_BASE_PATH in infra file (in `apps/api/infra/...`) and commit `yarn schemas` so this doesn't break main with missing config.